src/constants/chainInfo.ts

```diff
@@ -83,10 +53,7 @@ export const CHAIN_INFO: ChainInfoMap = {
     infoLink: 'https://info.uniswap.org/#/',
     label: 'Ethereum',
     logoUrl: ethereumLogoUrl,
-    addNetworkInfo: {
```

Comment:
is this `addNetworkInfo` for the interaction with metamask? Will this break if we remove it?